### DISCUSSION TITLE
大佬们如何直接访问云服务器而不是通过分享链接呢

### DISCUSSION THREAD

**Discussion by oldfishoh** [GaiZhenbiao/ChuanhuChatGPT#556] [Q&A] [answered]
目前是通过部署在云服务器上通过分享链接来使用的   但是这样有以下几个弊端

1. 分享链接无法和多账户功能同时使用（没有多账户功能 保存历史对话就很尴尬）

2. 分享链接的地址不固定且分享链接会失效

 所以想问下如何不用gradio来通过分享链接的方式代理访问我的云服务器 而是我可以直接访问我的云服务器

**Comment by GaiZhenbiao** [comment, marked as answer]
当然可以呀，Readme里就有写，看“在公网部署”部分

**Reply by oldfishoh** [reply]
感谢
